go/kbfs/libkbfs/conflict_resolver.go

```diff
@@ -3325,10 +3330,16 @@ func (cr *ConflictResolver) doResolve(ctx context.Context, ci conflictInput) {
 		fmt.Sprintf("%s %+v", cr.fbo.folderBranch, ci))
 	defer func() { cr.config.MaybeFinishTrace(ctx, err) }()
 
-	err = cr.recordStartResolve(ci)
+	var failCount int
+	failCount, err = cr.recordStartResolve(ci)
 	if err != nil {
 		cr.log.CWarningf(ctx,
 			"Could not record conflict resolution attempt: %v", err)
+	} else if failCount >= maxConflictResolutionAttempts {
+		cr.log.CWarningf(ctx,
```

Comment:
Did you intend to do something other than print a warning message here?  Like maybe returning?

Reply:
I did intend to return here.